<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FanjianBox</class>
 <widget class="QWidget" name="FanjianBox">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>25</number>
   </property>
   <property name="rightMargin">
    <number>25</number>
   </property>
   <property name="bottomMargin">
    <number>25</number>
   </property>
   <item>
    <widget class="QLabel" name="messageLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;font color=white&gt;&lt;b&gt;周瑜对你使用了反间，请选择花色&lt;/b&gt;&lt;/font&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="sgsui::GameButton" name="heartButton">
       <property name="text">
        <string>&lt;font color=white&gt;红桃&lt;/font&gt;&lt;font color=red&gt;♥&lt;/font&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="sgsui::GameButton" name="spadeButton">
       <property name="text">
        <string>&lt;font color=white&gt;黑桃&lt;/font&gt;&lt;font color=black&gt;♠&lt;/font&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="sgsui::GameButton" name="diamondButton">
       <property name="text">
        <string>&lt;font color=white&gt;方片&lt;/font&gt;&lt;font color=red&gt;♦&lt;/font&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="sgsui::GameButton" name="clubButton">
       <property name="text">
        <string>&lt;font color=white&gt;草花&lt;/font&gt;&lt;font color=black&gt;♣&lt;/font&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>sgsui::GameButton</class>
   <extends>QLabel</extends>
   <header>sgsui_GameButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
